### PR TITLE
[12.0][MIG] partner_contact_birthdate: Migration to 12.0

### PR DESCRIPTION
**Pull request by oihane**

### CODE REVIEW

**Review by alfredoavanzosc — APPROVED** [review]
:+1:

**Review by anajuaristi — APPROVED** [review]

**Review by andhit-r — APPROVED** [review]

File: partner_contact_birthdate/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2014-2015  Grupo ESOC <www.grupoesoc.es>
```
Comment (NL66278) [inline partner_contact_birthdate/__init__.py]: There should be no more copyright lines in __init__.py files.

File: partner_contact_birthdate/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+# © 2017-Apertoso N.V. (<http://www.apertoso.be>)
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (NL66278) [inline partner_contact_birthdate/__init__.py]: And the license lines should be changed to https:

File: partner_contact_birthdate/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2014-2015  Grupo ESOC <www.grupoesoc.es>
+# © 2017-Apertoso N.V. (<http://www.apertoso.be>)
```
Comment (NL66278) [inline partner_contact_birthdate/__manifest__.py]: The copyright lines that remain should use the word Copyright, not the symbol

File: partner_contact_birthdate/views/res_partner.xml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+<!-- License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+     © 2015 Grupo ESOC Ingeniería de Servicios, S.L.U. -->
+
+<odoo>
+    <record id="view_personal_information_birthdate_date" model="ir.ui.view">
+        <field name="name">Birthdate Date field</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="partner_contact_personal_information_page.personal_information"/>
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='personal_information_page']//group[@name='personal_information_group']">
```
Comment (NL66278) [inline partner_contact_birthdate/views/res_partner.xml]: An expression with ```//group[@name='personal_information_group']``` will be enough to reliably find the right spot to insert.

**Review by NL66278 — CHANGES_REQUESTED** [review]
Mostly OK, but needs some changes to adhere to current OCA conventions.

File: partner_contact_birthdate/__init__.py (hunk 1 above)
Comment (oihane) [reply partner_contact_birthdate/__init__.py]: DONE

File: partner_contact_birthdate/__init__.py (hunk 2 above)
Comment (oihane) [reply partner_contact_birthdate/__init__.py]: DONE

File: partner_contact_birthdate/__manifest__.py (hunk 3 above)
Comment (oihane) [reply partner_contact_birthdate/__manifest__.py]: DONE

File: partner_contact_birthdate/views/res_partner.xml (hunk 4 above)
Comment (oihane) [reply partner_contact_birthdate/views/res_partner.xml]: DONE

**Review by houssine78 — APPROVED** [review]

**Comment by houssine78** [comment]
@NL66278 I think that the author taken all your remarks into account. Could it be merged now ?

@pedrobaeza do you think it still miss something to be merged ?

Thanks you!

**Review by NL66278 — APPROVED** [review]

**Comment by houssine78** [comment]
@NL66278 thank you!

**Pull request merged**